src/handler.js

```diff
@@ -55,11 +80,27 @@ module.exports = options => {
     const method =
       req.method && req.method.toUpperCase && req.method.toUpperCase();
 
+    const components = parse(req.url);
+    const originalPath = components.path;
+    const params = match(originalPath);
+    const rest = params[0] || '';
+
+    let path = originalPath;
+
+    if (params !== false) {
+      path = `/${params.version}/${config.airtableBaseId}/${rest}`;
+    }
+
+    req.url = path;
+
+    const tableName = rest.split('?').shift();
```

Comment:
This doesn't work for every request. If you're making a PUT or DELETE, you'll include a resource ID, so `tableName` will include it. I think this works (although it might be safer to wrap in a try/catch):

```
const tableName = rest.split('?').shift().split('/')[0];
```